src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -36,8 +37,6 @@ static constexpr std::string_view ProfilesKey{ "profiles" };
 static constexpr std::string_view DefaultSettingsKey{ "defaults" };
 static constexpr std::string_view ProfilesListKey{ "list" };
 static constexpr std::string_view SchemesKey{ "schemes" };
-static constexpr std::string_view NameKey{ "name" };

```

Comment:
When I'm opening a file I let clang-tidy run and fix as many issues as is reasonable. And before opening a PR I often try to end up with more lines of code removed than added.

I can stop at any time to be honest, because I know it's a real bother for others (for which I'm very sorry). I'm personally convinced however that it's the right thing to do for a multitude of reasons.